pyOCD/target/target_kinetis.py

```diff
@@ -141,3 +164,20 @@ def massErase(self):
         else:
             logging.error("Failed to unlock. MDM status: 0x%x", val)
             return False
+
+    ## @brief Write the unsecure FSEC value to flash.
+    #
+    # This will prevent the device from being secured after the next reset.
+    def unlockFlash(self):
+        logging.info("Unlocking chip...")
+
+        # Init flash algorithm.
+        self.flash.init()
+
+        logging.info("Performing mass erase")
+        self.massErase()
```

Comment:
Mass erase will take care of writing 0xFE to the FCF to keep the part unlocked, so the other flash operations init and pageProgram shouldn't be needed.